www/analytics/last-score-by-activity-by-user/last-score-by-activity-by-user-cut-score1.php

```diff
@@ -0,0 +1,169 @@
+    .switch-text {
+        margin-left: 50px;
+        width: 100px;
+        display: inline-block
+    }
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td.lrn-report-clickable {
+        position: relative;
+        vertical-align: middle;
+    }
+
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td.lrn-report-clickable:hover::before {
+        content: '';
+        position: absolute;
+        top: 1px;
+        left: 1px;
+        right: 1px;
+        bottom: 1px;
+        box-shadow: 0 0 0 1px #696969;
+    }
+
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td.lrn-report-clickable:focus {
+        border: 2px solid #333333;
+    }
+</style>
+<!--visualization performance band fail styles-->
+<style id="band-fail-styles-last-score-by-activity">
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td[data-custom_performance="fail2"] {
+        background-color: #FBE3E3;
+        font-weight: 700;
+    }
+
+    .learnosity-report.lastscore-by-activity-by-user .lrn-report-table > tbody > tr > td[data-custom_performance="fail2"]::before {
+        content: '';
+        position: absolute;
+        top: 2px;
+        left: 2px;
+        right: 2px;
+        bottom: 2px;
+        box-shadow: 0 0 0 2px #DD002F;
+    }
+</style>
+
+<div class="lsbabu-report"  id="lsbabu-report-cutscore1" style="display: none">
+    <div class="form-check form-switch lrn-cut-score-switch" >
+
+    <input type="checkbox" id="highlight-band-fail1" class="switch-input highlight-band-fail1-switch1" checked >
+        <label for="highlight-band-fail1" class="switch-label"><span class="switch-text">Show failing</span></label>
+    </div>
+
+    <div class="learnosity-report" id="lastscore-by-activity-by-user-cutscore1"></div>
+</div>
+
+<script>
+    // last score by item by user report with cut score demo 1
+    const callbacksLastScoreByActivityCutScore1 = {
+        readyListener: function () {
+            const report = reportsAppLastScoreByActivityCutscore1.getReport('lastscore-by-activity-by-user-cutscore1');
+            report.on('load:data', function(data) {
+                console.log("cut score 1 load data: ", data);
+            });
+            report.on("click:score", function(data) {
+                console.log("click score: ", data);
+            });
+            report.on("click:user", function(data) {
+                console.log("click user: ", data);
+            });
+        },
+        errorListener: function (err) {
+            console.log(err);
+        },
+        scoreMutator: function(data) {
+            processLastScoreByActivityCutscore1(data);
+        }
+    };
+
+    function processLastScoreByActivityCutscore1(data) {
+        let performanceBand = 'none';
+        let voiceOverMessage = '';
+
+        if (data.percentageItemsCorrect <= 50) {
+            performanceBand = 'fail2';
```

Comment:
I think we can do `performanceBand = 'fail_band';` and change the other places accordingly. but if you have better idea, use yours.